src/components/Atoms/Alert/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color.js";
+import TEXT from "../../../variables/texts.js";
+import BREAKPOINT from "../../../variables/breakpoint.js";
+
+const Alert = ({ visible, errorText }) => {
+  return (
+    <StyledAlert visible={visible}>
+      <StyledAlertText>{errorText}</StyledAlertText>
+    </StyledAlert>
+  );
+};
+
+export default Alert;
+
+const StyledAlert = styled.div`
+  position: absolute;
+  width: 400px;
+  top: 80px;
+  border-radius: 4px;
+  background-color: ${COLOR.RED};
+  transition: all 0.5s ease 0s;
+  opacity: ${(visible) => (visible ? "0" : "1")};
```

Comment:
```suggestion
  opacity: ${(visible) => (visible ? "1" : "0")};
```
ここを変えたら動いたりしますかね？

Reply:
空入力でエラー文が出るようになりました！
そもそもvisibleがfalseになってないみたいで、三項演算子のtrue側しか機能していないようです。
<img width="618" alt="229309135-3984e391-272c-4419-8423-56d1b7e1c5e6" src="https://user-images.githubusercontent.com/71510837/229309979-ef300ba4-ba85-443f-9680-34abae4fa36c.png">
<img width="555" alt="229309610-455ea835-f7e4-4bef-8851-cc6e043d0e5d" src="https://user-images.githubusercontent.com/71510837/229309980-8799da87-d269-42e8-ad71-802e2de5f185.png">
<img width="514" alt="229309614-44c88fde-b3ad-47da-a960-0a4fa9425d53" src="https://user-images.githubusercontent.com/71510837/229309981-7ca63a28-d029-4144-b8a7-ab431badce86.png">

Reply:
```suggestion
  opacity: ${({visible}) => (visible ? "0" : "1")};
```
すみません、正しくはこうですね！

```suggestion
  opacity: ${(props) => (props.visible ? "0" : "1")};
```
もしくはこうです